### PR TITLE
feat(api): add API Key requirement to endpoints

### PR DESCRIPTION
**Pull request by Sven65**
More info at #CU-ht607d

### CODE REVIEW

**Comment by helsingborgsstad** [comment]
Task linked: [CU-ht607d Slå på api-key i serverless.yml på samtliga endpoints](https://app.clickup.com/t/ht607d)

**Review by TeddyAndersson — COMMENTED** [review]
Seems to work as expected, have you tested this with the app aswell?

File: services/bankid-api/serverless.yml [hunk 1]
```diff
@@ -68,25 +68,29 @@ functions:
...
           path: auth/bankid/collect
           method: post
+          private: true
           cors: true
   cancel:
     handler: lambdas/cancel.main
     events:
       - http:
           path: auth/bankid/cancel
           method: post
```
Comment (dannil76) [inline services/bankid-api/serverless.yml:94]: Is there a reason not to have the parameter authorizer defined?
See cases-api for reference.

Comment (Sven65) [reply services/bankid-api/serverless.yml:94]: I'm unsure whether or not it was needed. Seeing as the task only was for API Keys, and that we need to use the bankID to login, I opted to not include it.

**Comment by Sven65** [comment]
> Seems to work as expected, have you tested this with the app aswell?

I haven't tested it fully yet, but from the testing I have done, it works as intended.

**Pull request merged**